app/controllers/auth_controller.py

```diff
@@ -42,48 +42,56 @@
 
 router = APIRouter()
 
-
-oidc = OpenIdConnect(openIdConnectUrl=api_settings.oidc_metadata_url)
-
-public_key_cache: tuple[datetime, tuple[str, str]] | None = None
-
-
-@dataclass
-class AuthInfo:
-    user_id: str
-    roles: list[str]
-
-
-async def get_issuer_and_public_key() -> tuple[str, str]:
```

Comment:
Add "if not api_settings.use_authlib_oauth:"
The rest of the code does not change, it is only indented to the right.